lib/node_modules/@stdlib/stats/base/sstdevtk/lib/ndarray.js

```diff
@@ -33,22 +33,20 @@ var sqrt = require( '@stdlib/math/base/special/sqrt' );
 * @param {PositiveInteger} N - number of indexed elements
 * @param {number} correction - degrees of freedom adjustment
 * @param {Float32Array} x - input array
-* @param {integer} stride - stride length
-* @param {NonNegativeInteger} offset - starting index
+* @param {integer} strideX - stride length
+* @param {NonNegativeInteger} offsetX - starting index
 * @returns {number} standard deviation
 *
 * @example
 * var Float32Array = require( '@stdlib/array/float32' );
-* var floor = require( '@stdlib/math/base/special/floor' );
 *
 * var x = new Float32Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0 ] );
-* var N = floor( x.length / 2 );
 *
-* var v = sstdevtk( N, 1, x, 2, 1 );
+* var v = sstdevtk( 4, 1, x, 2, 1 );
 * // returns 2.5
 */
-function sstdevtk( N, correction, x, stride, offset ) {
-	return float64ToFloat32( sqrt( svariancetk( N, correction, x, stride, offset ) ) ); // eslint-disable-line max-len
+function sstdevtk( N, correction, x, strideX, offsetX ) {
+	return float64ToFloat32( sqrt( svariancetk( N, correction, x, strideX, offsetX ) ) ); // eslint-disable-line max-len
```

Comment:
```suggestion
	return sqrtf( svariancetk( N, correction, x, strideX, offsetX ) ); // eslint-disable-line max-len
```
where:
```javascript
var sqrtf = require( `@stdlib/math/base/special/sqrtf` );
```
will this not be cleaner? as both `sqrtf` and `svariancech` return single-precision floating point values, we can drop the `float64toFloat32` i believe

Reply:
cc @kgryte

Reply:
Yes, the proposed change makes sense!

Reply:
pushed a fix!